lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -453,7 +453,12 @@ int amf_smf_process_pdu_session_packet(
         return rc;
       }
 
-      smf_ctx->sst = msg->nssai.sst;
+      // NSSAI
+      if (msg->nssai.sst) {
+        smf_ctx->sst = msg->nssai.sst;
+      } else {
+        smf_ctx->sst = static_cast<uint8_t>(eMBB);
+      }
```

Comment:
you should be getting these values from AMF configurations. It does not make sense to respond sst value with whatever the UE is asking as we do not support URLLC, MIoT, etc.

Reply:
@ulaskozat, I fixed the second part: when the ue doesn't send nssai IE, I am using uswer configured sst value. But for the first part: when UE sends nssai IE, there is ongoing work for slicing feature support which will cover all the cases.